src/pytorch_lightning/strategies/ddp.py

```diff
@@ -57,7 +58,7 @@
 from pytorch_lightning.utilities.optimizer import optimizers_to_device
 from pytorch_lightning.utilities.rank_zero import rank_zero_info, rank_zero_only, rank_zero_warn
 from pytorch_lightning.utilities.seed import reset_seed
-from pytorch_lightning.utilities.types import STEP_OUTPUT
+from pytorch_lightning.utilities.types import STEP_OUTPUT, TestStep, TrainingStep, ValidationStep
```

Comment:
```suggestion
from pytorch_lightning.utilities.types import PredictStep, STEP_OUTPUT, TestStep, TrainingStep, ValidationStep
```

Reply:
fixed in [bf23171](https://github.com/Lightning-AI/lightning/pull/13885/commits/bf23171399ca748442c378b97251e64e1de3112c)